router/app-router.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const storage = require('../lib/storage.js');
+const Athlete = require('../model/athlete.js');
+
+module.exports = function(router) {
+  router.post('/api/athletes', function(req, res) {
+    if(!req.body.athlete_name || !req.body.sport) {
+      res.statusCode = 400;
+      res.end();
+      return;
+    }
+    let athlete = new Athlete(req.body);
+    storage.createItem('athletes', athlete)
+    .then(athlete => {
+      res.setHeader('Content-Type', 'application/json');
+      res.write(JSON.stringify(athlete));
+      res.end();
+    })
+    .catch(err => {
+      console.error(err);
+      res.statusCode = 500;
+      res.end();
+    });
+  });
+
+  router.get('/api/athletes', function(req, res){
+    let id = req.url.query.id;
+    if(!id) {
```

Comment:
Good job accounting for redirecting the lack of a query id to default to getting the array of all athlete IDs.